src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResourceIntTest.kt

```diff
@@ -955,6 +955,14 @@ class VisitResourceIntTest : IntegrationTestBase() {
 
         assertThat(updatedVisit.visitors).extracting<Long>(Visitor::personId)
           .containsExactlyInAnyOrder(neoAyomide.id, KashfAbidi.id)
+
+        // check visit order is updated with visitor changes
+        val visit = repository.getVisit(updatedVisit.visitId)
+        assertThat(visit.visitOrder).isNotNull
+        assertThat(visit.visitOrder?.visitors).extracting("person.id", "groupLeader").containsExactly(
+          tuple(neoAyomide.id, true),
+          tuple(KashfAbidi.id, false),
+        )
```

Comment:
maybe also test when you amend a visit that has no visit order (e.g a remand prisoner)?